src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -105,11 +107,28 @@ void tabulate() throws TabulationCancelledException {
     Logger.log(Level.INFO, "Starting tabulation session...");
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
     if (config != null && config.validate() && setUpLogging(config)) {
+      try {
+        Logger.log(Level.INFO, "Begin config file contents:");
```

Comment:
Is this the thing that got accidentally deleted in a merge? I swear you added this already.

Reply:
Yeah it's nutty.  This commit: "Merge pull request #314 from BrightSpots/bad_cvr_checks" a79cdd3371d94d7cd5af32d746420cfd052c810c done by the github web tool.  Both parent commits have this code in them so i don't understand why the merge did this.  I went through all the other changes and they check out.